vsc-error.yml

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright 2021, Magnus Feuer
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+
+#
+# Standard Vehicle Service Catalog error codes.
+#
+---
+
+#
+# The stdvsc namespace is composed of several standard files that all
+# install their entries in the same stdvsc namespace.
+#
+name: stdvsc
+major-version: 1
+minor-version: 0
+description: Standard error codes.
+
+enumerations:
+  # 
+  # Error enumueration
```

Comment:
Minor spelling error

Reply:
Fixed in https://github.com/COVESA/vehicle_service_catalog/pull/13/commits/9d7c4bbfc55a2e346355ef3de6a436f43e879006